standards-catalogue/config/tech-docs.yml

```diff
@@ -0,0 +1,26 @@
+# Host to use for canonical URL generation (without trailing slash)
+host: https://alphagov.github.io/data-standards-authority/
+
+# Header-related options
+show_govuk_logo: false
+service_name: Data Standards Authority
+service_link: https://alphagov.github.io/data-standards-authority/
+phase: Alpha
```

Comment:
Could we make this "Discovery" instead? carries less weight from a service assessment perspective.